contrib/storage-splunk/pom.xml

```diff
@@ -58,6 +58,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.github.ben-manes.caffeine</groupId>
+      <artifactId>caffeine</artifactId>
+      <version>2.9.3</version>
```

Comment:
Can we achieve the same thing using Guava's caching? The reason I ask is that we already have this insanely big dependency tree and Guava is already in it...

https://www.baeldung.com/guava-cache

Reply:
But so is caffeine now that I look! So I guess we can ignore this suggestion.

Reply:
@jnturton  Is that a +1?  I somehow broke the versioning when I rebased on the current master, but I'll fix before merging.

Reply:
Please lift the `caffeine.version` property from metastore/iceberg-metastore/pom.xml to the root pom and either
- add caffeine with `caffeine.version` to dependencyManagement in the root pom and remove version numbers here and in the Iceberg metastore, or
- make both this pom and Iceberg metastore pom specify `<version>${caeffine.version}</version>`.

So that we standardise the version of Caffeine that gets pulled in.